app/models/stacks_file.rb

```diff
@@ -15,12 +15,6 @@ def file_exist?
     path && File.exist?(path)
   end
 
-  def valid?
-    # FIXME: image_exist? is defined in app/models/concerns/djatoka_adapter.rb,
-    #  which is included in StacksImage, but is probably undefined here
-    image_exist?
-  end
```

Comment:
I'm 👌 w/ removing this method altogether.

It feels a bit odd to have a subclass define a method that the parent class does not have base behavior for (but that seems to be a pattern in the `StacksImage` class, so :shipit:)